lib/location/method_gnss.c

```diff
@@ -57,8 +57,21 @@ BUILD_ASSERT(
 #define AT_MDM_SLEEP_NOTIF_START "AT%%XMODEMSLEEP=1,%d,%d"
 #endif
 #if (defined(CONFIG_NRF_CLOUD_AGPS) || defined(CONFIG_NRF_CLOUD_PGPS))
+static int64_t agps_req_timestamp;
```

Comment:
Static into the function where this is used? I don't know if we do that because seeing sram consumption from the beginning of the file is not accurate but otherwise it's nicer.

Reply:
Maybe it's better inside the function, because it's only used in a single place.

Reply:
I modified the implementation and `agps_req_timestamp` is now accessed from two different places, so need to have it in the beginning of the file.